### PR TITLE
Update demo

### PR DESCRIPTION
**Pull request by wbloszyk**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -81,16 +81,19 @@
...
-        "behat/mink-extension": "^2.3.0",
-        "behat/mink-goutte-driver": "^1.2.1",
         "behat/mink": "^1.7.1",
         "behat/mink-browserkit-driver": "^1.3.2",
-        "behat/mink-selenium2-driver": "^1.3.1"
+        "behat/mink-extension": "^2.3.0",
+        "behat/mink-goutte-driver": "^1.2.1",
+        "behat/mink-selenium2-driver": "^1.3.1",
+        "phing/phing": "^2.16",
+        "phpunit/phpunit": "~4.2"
```
Comment (phansys) [inline composer.json]: Just for consistency:
```suggestion
        "phpunit/phpunit": "^4.2"
```

Comment (wbloszyk) [reply composer.json]: I will update it in next PR wldedicated for test

Comment (wbloszyk) [reply composer.json]: Also this change is from 
```json
"config": {
        "sort-packages": true
    },	    
```

**Comment by OskarStark** [comment]
Thank you 👋

**Pull request merged**